pkg/cloudprovider/provider/aws/provider.go

```diff
@@ -187,8 +189,21 @@ func getDefaultAMIID(client *ec2.EC2, os providerconfig.OperatingSystem, region
 				Name:   aws.String("root-device-type"),
 				Values: aws.StringSlice([]string{"ebs"}),
 			},
+			{
+				Name:   aws.String("architecture"),
+				Values: aws.StringSlice([]string{"x86_64"}),
```

Comment:
would it make sense to move the arch decision up to the caller and not hardcode it in getDefaultAMIID?

Reply:
The filters are just as hardcoded and right now we support amd64 only

Reply:
Just figured that there is no arm offering for it in marketplace anyways.